applications/StructuralMechanicsApplication/structural_mechanics_application_variables.cpp

```diff
@@ -158,4 +158,7 @@ KRATOS_CREATE_VARIABLE(MpcDataSharedPointerVectorType, MPC_DATA_CONTAINER)
 
 KRATOS_CREATE_VARIABLE(bool, INELASTIC_FLAG)
 KRATOS_CREATE_VARIABLE(double, INFINITY_YIELD_STRESS)
+
+// Response function variables
+KRATOS_CREATE_VARIABLE(double, RESPONSE_VALUE)
```

Comment:
What exactly are you doing with this variable?
I didn't understand it

Reply:
Until now, the gradient is stored in a nodal variable, e.g. ```SHAPE_SENSITIVITY```, but the response value was returned and not stored at all at the model part.
This concept is similar to e.g. the eigenvalue analysis, where the eigenvalues are stored as process info at the model part, and the eigenvectors in nodal variables.

It seemed just more convenient to store/treat both possible results of a response function in a similar way.